engine/src/flutter/impeller/entity/contents/filters/filter_contents.h

```diff
@@ -190,7 +190,29 @@ class FilterContents : public Contents {
       const Matrix& effect_transform,
       const Rect& output_limit) const = 0;
 
-  /// @brief  Converts zero or more filter inputs into a render instruction.
+  /// Applies the specific filter logic to the given inputs and returns an
+  /// Entity representing the filtered result.
+  ///
+  /// This is the primary method that subclasses must implement to define their
+  /// filtering behavior. It takes the results of evaluating the filter inputs
+  /// (as Snapshots) and produces a new Entity containing the filtered output.
+  ///
+  /// @param[in] inputs The evaluated inputs to the filter, typically as
+  /// Snapshots.
+  /// @param[in] renderer The content context providing rendering resources.
+  /// @param[in] entity The entity applying this filter, providing transform,
+  /// blend mode, and other context.
+  /// @param[in] effect_transform An additional transform applied after the
+  /// entity's transform, often used in subpass scenarios.
+  /// @param[in] coverage The calculated coverage area of the filter's output
```

Comment:
Would "coverage" typically have been calculated by the FilterContents itself, as opposed to "coverage_hint" which is imposed by the surrounding context in which it's being rendered (such as a clip operation)?

In other words, "here is the coverage you calculated so you don't have to calculate it again should you need it"?

Reply:
Yea exactly, if you look at `FilterContents::GetEntity` you can see that.  I agree it's confusing, this is a private method so it makes a bit more sense.